client/src/pages/lessons/edit.tsx

```diff
@@ -278,8 +293,9 @@ const LessonEdit = (props: { search: LessonEditSearch }) => {
         if (lesson) {
           setLesson(lesson);
         }
+
         if (lessonId !== lesson.lessonId) {
-          navigate(`/lessons/edit/?lessonId=${lesson.lessonId}`);
+          window.location.href = `/lessons/edit?lessonId=${lesson.lessonId}`;
```

Comment:
not sure if there's a problem with `navigate` but there must be a correct React/Gatsby way to change page that isn't direct set of `window.location.href`?